src/MToken.sol

```diff
@@ -78,6 +79,17 @@ contract MToken is IMToken, ContinuousIndexing, ERC20Extended {
 
     /// @inheritdoc IMToken
     function burn(address account_, uint256 amount_) external onlyPortal {
+        uint256 spenderAllowance_ = allowance[account_][msg.sender];
```

Comment:
what happens if an account is a `msg.sender` itself? It won't have an allowance set, and won't be able to proceed with the call.

Reply:
resolved in conversation. This allowance feature is only for Portal